src/main/java/com/chartiq/finsemble/example/JavaExample.java

```diff
@@ -378,6 +391,29 @@ private void setFormEnable(boolean enabled) {
         dockingCheckbox.setDisable(!enabled);
     }
 
+    private void handleGetComponentStateCb(JSONObject err, JSONObject res) {
+        if(err!=null){
+            fsbl.getClients().getLogger().error(err.toString());
+        }else{
+            //Set subscribe linker channel
+            JSONArray channelToLink = res.getJSONArray("Finsemble_Linker");
+            for(int i=0;i<channelToLink.length();i++){
+                Button tempBtn = (Button) window.getScene().lookup("#"+channelToLink.getString(i)+"Button");
+                Platform.runLater(() -> tempBtn.fire());
+                fsbl.getClients().getLogger().log(channelToLink.getString(i));
+            }
```

Comment:
You need to check whether `res` has a property "Finsemble_Linker" before you try to get the JSONArray. 
```suggestion
            if (res.has("Finsemble_Linker")) {
                final JSONArray channelToLink = res.getJSONArray("Finsemble_Linker");
                for (int i=0; i< channelToLink.length(); i++) {
                    final Button tempBtn = (Button) window.getScene().lookup("#" + channelToLink.getString(i) + "Button");
                    Platform.runLater(() -> tempBtn.fire());
                    fsbl.getClients().getLogger().log(channelToLink.getString(i));
                }
            }
```